### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by jaapio**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,65 +19,62 @@ DocBlock comments to generate a complete set of API Documentation.
 Inspired by phpDocumentor 1 and JavaDoc it continues to innovate and is up to date
 with the latest technologies and PHP language features.
 
-Features
---------
+phpDocumentor v2 (stable)
+--------------
+The current stable version of phpdocumentor is v2.9. Which can be used to generate documentation for php
```
Comment (ashnazg) [inline README.md]: s/is v2.9. Which/is v2.9, which/

File: README.md [hunk 2]
```diff
@@ -19,65 +19,62 @@ DocBlock comments to generate a complete set of API Documentation.
...
-* *Low memory usage*, peak memory usage for small projects is less than 20MB, medium projects 40MB and large frameworks 100MB.
-* *Incremental parsing*, if you kept the Structure file from a previous run you get an additional performance boost of up
-  to 80% on top of the mentioned processing speed above.
-* *Easy template building*, if you want to make a branding you only have to call 1 task and edit 3 files.
-* *Command-line compatibility with phpDocumentor 1*, phpDocumentor 2 is an application in its own right but the
-  basic phpDocumentor 1 arguments, such as --directory, --file and --target, have been adopted.
-* *Two-step process*, phpDocumentor first generates a cache with your application structure before creating the output.
-  If you'd like you can use that to power your own tools or formatters!
+**Note:**
+The phpDocumentor team is currently working on a new phpDocumentor generation. Which will have full support for the latest features in PHP, so you can take advantage of all native type hints. v3 is still under heavy development
```
Comment (ashnazg) [inline README.md]: s/generation. Which/generation. It/

File: README.md [hunk 3]
```diff
@@ -19,65 +19,62 @@ DocBlock comments to generate a complete set of API Documentation.
...
+and is not stable enough to use in a daily development flow. 
 
-*Please note* that phpDocumentor 3 is still under heavy development. We aim to add all features needed to have full support
-for php 7+. But at this moment that is not the case.
+### nightly builds
+
+Phpdocumentor doesn't have a nightly release. But during each pipeline, a [phar] artifact is built. If you want
```
Comment (ashnazg) [inline README.md]: s/release. But/release. However,/

File: README.md [hunk 4]
```diff
@@ -19,65 +19,62 @@ DocBlock comments to generate a complete set of API Documentation.
...
+**Note:**
+The phpDocumentor team is currently working on a new phpDocumentor generation. Which will have full support for the latest features in PHP, so you can take advantage of all native type hints. v3 is still under heavy development
+and is not stable enough to use in a daily development flow. 
 
-*Please note* that phpDocumentor 3 is still under heavy development. We aim to add all features needed to have full support
-for php 7+. But at this moment that is not the case.
+### nightly builds
+
+Phpdocumentor doesn't have a nightly release. But during each pipeline, a [phar] artifact is built. If you want 
+to test the bleeding edge version of phpDocumentor have a look in the [actions](https://github.com/phpDocumentor/phpDocumentor/actions?query=workflow%3A%22Qa+workflow%22+is%3Asuccess) section of this repository.
```
Comment (ashnazg) [inline README.md]: s/of phpDocumentor have/of phpDocumentor, have/

File: README.md [hunk 5]
```diff
@@ -19,65 +19,62 @@ DocBlock comments to generate a complete set of API Documentation.
...
+Each successful QA workflow has a download at the right upper corner.
 
-Requirements
+Installation
 ------------
+phpDocumentor requires php 7.2 or higher to run. However, code of lower php versions can be analyzed.
 
-phpDocumentor requires the following:
+All templates provided with phpDocumentor so have support for Class diagrams based on the read code base. This will
```
Comment (ashnazg) [inline README.md]: s/ so//

File: README.md [hunk 6]
```diff
@@ -19,65 +19,62 @@ DocBlock comments to generate a complete set of API Documentation.
...
 
-Please see the documentation about creating your own templates for more information.
+### Using Phive
 
-Installation
-------------
+`$ phive install --force-accept-unsigned phpDocumentor`
 
-There are 3 ways to install phpDocumentor:
+For more information about phive have a look at their [website](https://phar.io/)
```
Comment (ashnazg) [inline README.md]: s/)/)./

File: README.md [hunk 7]
```diff
@@ -19,65 +19,62 @@ DocBlock comments to generate a complete set of API Documentation.
...
-Please see the documentation about creating your own templates for more information.
+### Using Phive
 
-Installation
-------------
+`$ phive install --force-accept-unsigned phpDocumentor`
 
-There are 3 ways to install phpDocumentor:
+For more information about phive have a look at their [website](https://phar.io/) 
+now you have phpDocumentor installed and it can be executed directly.
```
Comment (ashnazg) [inline README.md]: s/now/Now/

File: README.md [hunk 8]
```diff
@@ -19,65 +19,62 @@ DocBlock comments to generate a complete set of API Documentation.
...
-2. Via [Docker](https://hub.docker.com/r/phpdoc/phpdoc/)
-3. Via [Composer](https://getcomposer.org)
+`php tools/phpDocumentor`
 
 ### Using the PHAR
 
 1. Download the phar file from https://github.com/phpDocumentor/phpDocumentor/releases
-2. ???
-3. Profit!
+2. Profit!
```
Comment (ashnazg) [inline README.md]: I'd drop the old "profit" meme, and instead show the explicit syntax to run the phar.

**Comment by jaapio** [comment]
> English grammar review always feels very nitpicky thinking
Please do, because you are native English, which I'm not. So I can only learn from your feedback :-D

File: README.md (hunk 3 above)
Comment (ashnazg) [reply README.md]: Still needs a comma right after `However`-- `However, during each pipeline, `

... unless you want to adjust the clause sequencing -- `However, a [phar] artifact is built during each pipeline.`

**Pull request merged**